recipes/vvv2_display/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "vvv2_display" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "85d0b538d0e349474857dee9e048cd29a5e5f028f2d610003945cc1ec1709f53" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ANSES-Ploufragan/vvv2_display/archive/refs/tags/{{version}}.tar.gz
+  sha256: {{ sha256 }}
+ 
+build:
+  run_exports:
+  - {{ pin_subpackage("vvv2_display", max_pin="x.x") }}
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv 
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+    - setuptools >=57.4.0
+  run:
+    - pip
+    - python >=3.9
+    - r-ggplot2 >=3.4.4
+    - r-gridextra >=2.3
+    - r-cowplot >=1.1.1
+    - r-stringr >=1.5.1
+    - r-jsonlite >=1.8.8
+    - pysam >=0.19.1
+    - numpy >=1.23.3
+
+test:
+  commands:
+    - vvv2_display.py -h
+
+about:
+  home: https://github.com/ANSES_Ploufragan/vvv2_display/
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
+  summary: "Creates png image file with all [vardict] variants proportions alongside genome/assembly with annotations from [vadr]."
+  maintainer: FTouzain
```

Comment:
this should go into an `extras:` section.

Reply:
Current version has already maintainer in the `extra:` section:
```
about:
  home: https://github.com/ANSES-Ploufragan/vvv2_display/
  license: GPL-3.0-only
  license_family: GPL3
  license_file: LICENSE
  summary: "Creates png image file with all [vardict] variants proportions along genome/assembly with annotations from [vadr]."
  dev_url: https://github.com/ANSES-Ploufragan/vvv2_display

extra:
  recipe-maintainers:
    - FTouzain
```
I think it does not concerns license_family, license_file, and summary, isn't it? (I check on spades recipes to verify).

Many thanks for your feedback

Reply:
Corrected. I believed it was done (probably because working on 2 computers, but I was sure to have pushed... not matter). Sorry. It must be ok now.